### ISSUE TITLE
No WebPageTestServerController found

### ISSUE THREAD

**Issue by coderDem** [IteraSpeed/OpenSpeedMonitor#5]
Hello,

I try to set the WPT-URL for the measurement,
but can't find under Go to the WebPageTestServerController setting.
Must I login to see this setting?

best regards

**Comment by nilskuhn** [comment]
Hi,
yes, you have to be logged in to set base url of your WebPagetestserver/s.
If you run OpenSpeedMonitor in development environment for the first time admin and root users get created with passwords 'admin' and 'root'.
In production mode you have to set credentials for initial application users in external configuration file. See sample config file for details: https://github.com/IteraSpeed/OpenSpeedMonitor/blob/master/grails-app/conf/OpenSpeedMonitor-config.groovy.sample
Regards, Nils

**Comment by coderDem** [comment]
Hello,

I changed the OpenSpeedMonitor-config.groovy.sample to OpenSpeedMonitor-config.groovy
Then I changed the entries for username and password like this:

``grails.de.iteratec.osm.security.initialOsmAdminUser.username='[admin]'
grails.de.iteratec.osm.security.initialOsmAdminUser.password='[password]'
grails.de.iteratec.osm.security.initialOsmRootUser.username='[root]'
grails.de.iteratec.osm.security.initialOsmRootUser.password='[password]'
``

Then I generated the War with: ``grailsw.bat war`` 
And started the War with: ``grailsw.bat prod run-war``
I tried to log in with username: ``admin`` and password: ``password``.
But I get: ``"Sorry, we were not able to find a user with that username and password."``

What do I wrong, please help?

best regards

**Comment by Trosky** [comment]
If you want to use `admin` and `password ` please configure it as:

`grails.de.iteratec.osm.security.initialOsmAdminUser.username='admin'`
`grails.de.iteratec.osm.security.initialOsmAdminUser.password='password'`

The `[ ]` are only placeholders...

**Comment by nilskuhn** [comment]
Trosky's rigth, the brackets determine placeholders.

BTW

You don't have to run 

    grailsw.bat war

in front of

    grailsw.bat run-war

Second command runs application in embedded container. The _war_ command creates a war in folder _target_ for deployment in an external servlet container like tomcat.

**Comment by coderDem** [comment]
Hello,

thank you for the help. I could login with the root credentials. username: root and password: password.
Now I tried to add a entry for a new  Webpagetestserver. My Webpagetestserver is on localhost.
So I set the ``Base URL to:	http://127.0.0.1`` 
``But if I try to test the entry, I get this error message:
Executing action [loadLocations] of controller [de.iteratec.osm.measurement.environment.WebPageTestServerController] caused exception Executing action [loadLocations] of controller [de.iteratec.osm.measurement.environment.WebPageTestServerController] caused exception WebPageTestServerController has problem at line 36 ``

It is not possible to set the Base URL to adress like: http://127.0.0.1 ?
